### DISCUSSION TITLE
Anyway to pass parameters when creating IOCs?

### DISCUSSION THREAD

**Discussion by Qbert777** [CrowdStrike/falconpy#180] [Q&A] [answered]
When calling 'falcon.indicator_create_v1' there seems to only be a position for body:
i.e. response = falcon.indicator_create_v1(body=BODY)

Are there any way to pass params such as:
params = (
('retrodetections',"false"),
('ignore_warnings',"true")
)
falcon.indicator_create_v1(body=BODY, params=params)

P.S. - Obviously I tried this and it didn't work. ;-)

**Comment by jshcodes** [comment, marked as answer]
Hi @Qbert777 -

This is a bug! :bug: (The query string payload isn't addressed in the service class method.) 

I'm converting this into an issue now, we will submit a PR to resolve this asap.

In the interim, you should still be able to do this via the Uber class, something along the lines of:

```python
from falconpy.api_complete import APIHarness as Uber

falcon = Uber(creds=creds)

result = falcon.command("indicator_create_v1", body=BODY, parameters={"ignore_warnings": True})
print(result)
```

**Reply by Qbert777** [reply]
That works perfectly.  Thanks for the quick response!

**Comment by jshcodes** [comment]
Hi @Qbert777 -

After upgrading to [version 0.5.1](https://github.com/CrowdStrike/falconpy/releases/tag/v0.5.1), you should be able to use the Service Class to perform this action properly.  

Let us know if you have any problems.  😃
